pdf/invoice.product-tab.tpl

```diff
@@ -138,15 +138,11 @@
             {/if}
           </td>
 
-          <td class="center">
-            ({if $customization.quantity == 0}1{else}{$customization.quantity}{/if})
-          </td>
+					<td class="center"></td>
 
           {assign var=end value=($layout._colCount-3)}
           {for $var=0 to $end}
-            <td class="center">
-              --
-            </td>
+						<td class="center"></td>
```

Comment:
Just to be a bit picky, can you fix the indentation here, please?

Reply:
Tabs have slipped in there somehow, fixed in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/b8e71593cf9c85e380de95b57cb6bea1d3c94927